config/examples/Creality/Ender-3/BigTreeTech SKR Mini E3 3.0/Configuration_adv.h

```diff
@@ -1779,7 +1779,7 @@
 
   //#define SD_REPRINT_LAST_SELECTED_FILE // On print completion open the LCD Menu and select the same file
 
-  #define AUTO_REPORT_SD_STATUS           // Auto-report media status with 'M27 S<seconds>'
+  //#define AUTO_REPORT_SD_STATUS         // Auto-report media status with 'M27 S<seconds>'
```

Comment:
This is another one of those options that can help reduce overhead, so no harm in keeping it enabled if it fits.